Classes/Misc_PickupSpawner.uc

```diff
@@ -0,0 +1,14 @@
+class Misc_PickupSpawner extends Actor
+    abstract
+    notplaceable;
+
```

Comment:
You could probably derive from Object instead of Actor since it's not really working as an actor.  Again not a big deal.